composer.json

```diff
@@ -169,7 +169,7 @@
     "plugins/gravityformshubspot": "*",
     "plugins/gravityformsquiz": "*",
     "plugins/gravityformssurvey": "*",
-    "wpackagist-plugin/post-type-switcher": "3.3.0",
+    "wpackagist-plugin/post-type-switcher": "4.0.*",
```

Comment:
Looking at the changelog of previous releases, I think it's safe to use `4.*` here.